css/ampshared.css

```diff
@@ -237,7 +239,8 @@ i-amphtml-sizer {
 
 /* Hide all children of non-container elements. */
 .i-amphtml-notbuilt:not(.i-amphtml-layout-container) > * ,
-[layout]:not([layout=container]):not(.i-amphtml-element) > *
+[layout]:not([layout=container]):not(.i-amphtml-element) > *,
```

Comment:
I realized this also doesn't account for implicit container layout. Is there even a CSS selector that would match?

Reply:
What defines implicit container? Is it just a lack of `width` and `height` and `layout`?

Reply:
https://github.com/ampproject/amphtml/blob/67815dc7b6c1f434a09fcd52227a01f7b0b2c728/src/layout.js#L429-L442

Reply:
I don't believe we can do this without `:not(:resolved)`